ui/create_restore_page.go

```diff
@@ -0,0 +1,555 @@
+		keyEvent:                    common.keyEvents,
+		errChan:                     common.errorChannels[PageCreateRestore],
+		toCreateWalletWidget:        new(widget.Button),
+		backCreateRestoreWidget:     new(widget.Button),
+		showRestoreWidget:           new(widget.Button),
+		hideRestoreWidget:           new(widget.Button),
+		showPasswordModalWidget:     new(widget.Button),
+		hidePasswordModalWidget:     new(widget.Button),
+		spendingPasswordWidget:      new(widget.Editor),
+		matchSpendingPasswordWidget: new(widget.Editor),
+		addWalletWidget:             new(widget.Button),
+
+		errLabel:              common.theme.Body1(""),
+		spendingPassword:      common.theme.Editor("Enter password"),
+		matchSpendingPassword: common.theme.Editor("Enter password again"),
+		addWallet:             common.theme.Button("create wallet"),
+	}
+
+	pg.matchSpendingPasswordWidget.SingleLine = true
+	pg.create = common.theme.Button("create wallet")
+	pg.showPasswordModal = common.theme.Button("proceed")
+	pg.showRestoreWallet = common.theme.Button("Restore an existing wallet")
+	pg.showRestoreWallet.Background = color.RGBA{}
+	pg.showRestoreWallet.Color = common.theme.Color.Hint
+
+	pg.closeCreateRestore = common.theme.IconButton(mustIcon(decredmaterial.NewIcon(icons.NavigationArrowBack)))
+	pg.closeCreateRestore.Background = color.RGBA{}
+	pg.closeCreateRestore.Color = common.theme.Color.Hint
+
+	pg.hideRestoreWallet = common.theme.IconButton(mustIcon(decredmaterial.NewIcon(icons.NavigationArrowBack)))
+	pg.hideRestoreWallet.Background = color.RGBA{}
+	pg.hideRestoreWallet.Color = common.theme.Color.Hint
+
+	pg.hidePasswordModal = common.theme.Button("cancel")
+	pg.hidePasswordModal.Color = common.theme.Color.Danger
+	pg.hidePasswordModal.Background = color.RGBA{R: 238, G: 238, B: 238, A: 255}
+
+	pg.errLabel.Color = pg.theme.Color.Danger
+	for i := 0; i <= 32; i++ {
+		pg.seedEditors = append(pg.seedEditors, common.theme.Editor(fmt.Sprintf("Input word %d...", i+1)))
+		pg.seedEditorWidgets.focusIndex = -1
+		pg.seedEditorWidgets.editors = append(pg.seedEditorWidgets.editors, widget.Editor{SingleLine: true, Submit: true})
+	}
+
+	pg.seedListLeft, pg.seedListRight = &layout.List{Axis: layout.Vertical}, &layout.List{Axis: layout.Vertical}
+
+	return func() {
+		pg.layout(common)
+		pg.handle(common)
+	}
+}
+
+func (pg *createRestore) layout(common pageCommon) {
+	pg.theme.Surface(pg.gtx, func() {
+		toMax(pg.gtx)
+		pd := unit.Dp(15)
+		layout.Flex{Axis: layout.Vertical, Spacing: layout.SpaceBetween}.Layout(pg.gtx,
+			layout.Flexed(1, func() {
+				layout.Inset{Top: pd, Left: pd, Right: pd}.Layout(pg.gtx, func() {
+					layout.Flex{Axis: layout.Vertical}.Layout(pg.gtx,
+						layout.Flexed(1, func() {
+							if common.states.creating {
+								pg.processing()()
+							} else if pg.showRestore {
+								pg.Restore()()
+							} else {
+								pg.mainContent()()
+							}
+						}),
+					)
+				})
+			}),
+		)
+		if pg.showPassword {
```

Comment:
This is supposed to show the create modal i believe. it should be `showCreat` instead.